src/System.Net.Http.Rtc/src/Configurations.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcore50-Windows_NT;
```

Comment:
Yes, this assembly is shipped in its own package and netcore50(uap10.0) is the latest uap configuration that we had for it.